### PR TITLE
Add sender docproperties when creating document from template.

### PR DESCRIPTION
**Pull request by njohner**
With this PR we add support for defining a sender when creating a document from a template. This will add the corresponding docproperties to the created document. We of course also add support for defining a sender in the API.

For [CA-4429]

## Checklist
- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)
- API change:
  - [x] Documentation is updated
  - [x] API Changelog entry (see [guide](https://4teamwork.atlassian.net/wiki/spaces/4TEAM/pages/451248812/API+Changelog+Guidelines))
- New translations
  - [x] All msg-strings are unicode

[CA-4429]: https://4teamwork.atlassian.net/browse/CA-4429?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: opengever/dossier/templatefolder/form.py [hunk 1]
```diff
@@ -324,10 +356,58 @@ class ISelectRecipientAddress(model.Schema):
         ))
 
 
-class SelectAddressWizardStep(
+class ISelectSenderAddress(model.Schema):
```
Comment (phgross) [inline opengever/dossier/templatefolder/form.py:367]: Do we really want to add support to select the sender in the old UI or is that  necessary to add support to the API?

File: opengever/dossier/templatefolder/form.py [hunk 2]
```diff
@@ -97,6 +97,13 @@ class ICreateDocumentFromTemplate(model.Schema):
         required=False,
     )
 
+    form.widget('sender', KeywordFieldWidget, async=True)
+    sender = schema.Choice(
+        title=_(u'label_sender', default=u'Sender'),
+        source=ContactsSourceBinder(),
```
Comment (phgross) [inline opengever/dossier/templatefolder/form.py:103]: Does ContactsSourceBinder include users?

**Review by phgross — APPROVED** [review]
👍 LGTM

File: opengever/dossier/templatefolder/form.py (hunk 1 above)
Comment (njohner) [reply opengever/dossier/templatefolder/form.py:367]: No I guess it was not necessary 😅. But I'd say now that it's done, I'd leave it in

File: opengever/dossier/templatefolder/form.py (hunk 2 above)
Comment (njohner) [reply opengever/dossier/templatefolder/form.py:103]: yes that includes ogds users

File: opengever/dossier/templatefolder/form.py (hunk 1 above)
Comment (phgross) [reply opengever/dossier/templatefolder/form.py:367]: yes sure

**Pull request merged**